.github/ISSUE_TEMPLATE/support_request.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Support request
+about: If you have a support contract with Google, please create an issue in the Google Cloud Support console.
```

Comment:
Is this the correct `about` text? This doesn't seem to describe what a "Support request" encompasses; unlike the other two issue types.